content/donate/index.md

```diff
@@ -1,59 +1,47 @@
 ---
-title: Feed your developers
-description: Donate to support the development of our free application
+title: Donate to support the development of Organic Maps
+description: Your money pays for all project-related expenses and motivates us to improve Organic Maps.
```

Comment:
@MetehanOzyurek can you please take a look at the updated main and donation pages here (in English), and update the Turkish translation accordingly?
@Genio2003 can you please do the same with the Italian translation?

Reply:
OK, I'll check asap